Makefile.js

```diff
@@ -58,9 +58,13 @@ var NODE = "node ", // intentional extra space
     JS_FILES = find("lib/").filter(fileType("js")).join(" "),
     JSON_FILES = find("conf/").filter(fileType("json")).join(" ") + " .eslintrc",
     MARKDOWN_FILES_ARRAY = find("docs/").concat(ls(".")).filter(fileType("md")),
-    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" ");
+    TEST_FILES = find("tests/lib/").filter(fileType("js")).join(" "),
     /* eslint-enable no-use-before-define */
 
+    // Regex
+    TAG_REGEX = /^[Fix\:|Update\:|Breaking\:|Docs\:|Build\:|New\:|Upgrade\:]/,
+    ISSUE_REGEX = /[\(fixes|\(refs] #\d+\)$/;
```

Comment:
I prefer `/\((?:fixes|refs) #\d+(?:, (?:fixes|refs) #\d+)*\)$/` for commit for multiple issues.